apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloServerInterceptor.java

```diff
@@ -134,16 +123,24 @@ private Call httpCall(Operation operation) throws IOException {
     return httpCallFactory.newCall(requestBuilder.build());
   }
 
-  private RequestBody httpRequestBody(Operation operation) throws IOException {
+  private RequestBody httpRequestBody(Operation operation) {
     JsonAdapter<Operation> adapter = new OperationJsonAdapter(moshi);
     Buffer buffer = new Buffer();
-    adapter.toJson(buffer, operation);
+    try {
+      adapter.toJson(buffer, operation);
+    } catch (IOException e) {
+      throw new AssertionError(e);
+    }
     return RequestBody.create(MEDIA_TYPE, buffer.readByteString());
   }
 
-  public static String cacheKey(RequestBody requestBody) throws IOException {
+  public static String cacheKey(RequestBody requestBody) {
     Buffer hashBuffer = new Buffer();
-    requestBody.writeTo(hashBuffer);
+    try {
+      requestBody.writeTo(hashBuffer);
+    } catch (IOException e) {
```

Comment:
Could we log in case of error, I know it should never happen but lets add logs here:
`logger.e(e, "Failed to prepare http call");`